app/models/school.rb

```diff
@@ -115,8 +115,9 @@ class School < ApplicationRecord
   extend FriendlyId
   include EnergyTariffHolder
   include ParentMeterAttributeHolder
-  include EnumDataSharing
   include MailchimpUpdateable
+  include Enums::DataSharing
+  include Enums::SchoolType
```

Comment:
Moved shared enums to a module and added school type this way. This was when I thought we were using the school type enum from school in the case study module. Though it would be a shame to move it back now it's not needed.